test/utils.js

```diff
@@ -9,3 +12,12 @@ exports.signInData = {
   email: 'joedoe@wolox.co',
   password: 'holamundo2019'
 };
+
+factory.define('user', User, {
```

Comment:
you are repeating `user` name rigth?
I find it dangerous for the code maintainability